--- Makefile.orig	1970-01-01 02:00:00.000000000 +0200
+++ Makefile	2026-07-13 14:50:51.346412821 +0300
@@ -136,13 +136,17 @@
 		DEBUG      = 
 		TIME       =
 		OPENMP     = 
-		INCMODULES = -J /users/$$USER/EasyBuild/SW/LUMI-23.09/G/hipfort/0.4-0-cpeCray-23.09-rocm5.2/include/hipfort/amdgcn
+		INCMODULES = -J ${EBU_USER_PREFIX}/SW/LUMI-25.03/G/hipfort/0.4-0-cpeCray-25.03/include/hipfort/amdgcn \
+			     -L ${EBU_USER_PREFIX}/SW/LUMI-25.03/G/hipfort/0.4-0-cpeCray-25.03/lib \
+			     -L /opt/rocm/lib/
 	else ifeq ($(mode),gpu_profiling)
 		OPT        = -O3 -hfp3 -hacc -DAMD -h acc_model=auto_async_none:fast_addr:deep_copy -eZ -m4 -lhipfort-amdgcn -lrocrand -lroctx64 $(INCMODULES)
 		DEBUG      = -g
 		TIME       = -DROCTX
 		OPENMP     = 
-		INCMODULES = -J /users/$$USER/EasyBuild/SW/LUMI-23.09/G/hipfort/0.4-0-cpeCray-23.09-rocm5.2/include/hipfort/amdgcn
+		INCMODULES = -J ${EBU_USER_PREFIX}/SW/LUMI-25.03/G/hipfort/0.4-0-cpeCray-25.03/include/hipfort/amdgcn \
+			     -L ${EBU_USER_PREFIX}/SW/LUMI-25.03/G/hipfort/0.4-0-cpeCray-25.03/lib \
+			     -L /opt/rocm/lib/
 	endif
 endif
 
@@ -325,7 +329,7 @@
 $(OBJDIR)bc_module.o			 	: $(SRCDIR)bc_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o mpi_module.o fluid_functions_module.o inflow_module.o real_to_integer_module.o mpi_comm_module.o)
 $(OBJDIR)rhs_module.o			 	: $(SRCDIR)rhs_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o mpi_module.o advection_module.o viscous_module.o integration_module.o sgs_module.o)
 $(OBJDIR)advection_module.o			: $(SRCDIR)advection_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o mpi_module.o flux_module.o inflow_module.o nvtx.o roctx.o nscbc_boundary_conditions.o)
-$(OBJDIR)shock_detection_module.o		: $(SRCDIR)shock_detection_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o fluid_functions_module.o mpi_comm_module.o)
+$(OBJDIR)shock_detection_module.o		: $(SRCDIR)shock_detection_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o fluid_functions_module.o mpi_comm_module.o  profiling_module.o)
 $(OBJDIR)viscous_module.o			: $(SRCDIR)viscous_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o mpi_module.o nvtx.o roctx.o)
 $(OBJDIR)flux_module.o				: $(SRCDIR)flux_module.f90 $(addprefix $(OBJDIR), parameters_module.o storage_module.o mpi_module.o)
 $(OBJDIR)inflow_module.o			: $(SRCDIR)inflow_module.f90 $(addprefix $(OBJDIR), storage_module.o fluid_functions_module.o math_tools_module.o df_module.o)
